Source/Serilog.Exceptions/LoggerEnrichmentConfigurationExtensions.cs

```diff
@@ -22,9 +23,10 @@ public static class LoggerEnrichmentConfigurationExtensions
 
         public static Serilog.LoggerConfiguration WithExceptionDetails(
             this LoggerEnrichmentConfiguration loggerEnrichmentConfiguration,
-            IEnumerable<IExceptionDestructurer> destructurers)
+            IEnumerable<IExceptionDestructurer> destructurers,
+            IExceptionPropertyFilter filter)
```

Comment:
Perhaps this should be an IEnumerable<IExceptionPropertyFilter> too, or have an overload of the method which accepts either a single or an enumerable. That'd mean consumers aren't forced to implement their own composite IExceptionPropertyFilter implementations in order to chain them up.

Reply:
I'm wondering the same.

Reply:
I'm wondering the same. The alternative would be providing some kind of filter that accepts and uses multiple filters itself.

Reply:
Because a single interface is so flexible, I implemented CompositeExceptionPropertyFilter.

Reply:
I've seen it done both ways in public API's. If it was `IEnumerable`, we could also add a `params` constructor. On the other had `CompositeExceptionPropertyFilter` makes it pay for play. I slightly prefer the former.